src/Serilog.Ui.Web/Extensions/UiOptions.cs

```diff
@@ -14,10 +14,22 @@ public class UiOptions
         /// <value> The route prefix. </value>
         public string RoutePrefix { get; set; } = "serilog-ui";
 
+        /// <summary>
+        ///     Gets or sets the URL for the home button
+        /// </summary>
+        /// <value> The URL for the home button. </value>
+        public string HomeUrl { get; set; } = "/";
+
         /// <summary>
         ///     Gets or sets the type of the authentication.
         /// </summary>
         /// <value> The type of the authentication. </value>
         internal string AuthType { get; set; }
+
+        /// <summary>
+        ///     Gets or sets the head content, a string that will be placed in the &lt;head&gt; of the index.html
+        /// </summary>
+        /// <value> The head content. </value>
+        internal string HeadContent { get; set; } = "";
```

Comment:
```suggestion
        internal string HeadContent { get; set; } = string.Empty;
```
sry, just had a couple of sonar-intensive weeks... 🤣

Reply:
Haha - i feel you. but TBH it used to be so that in the past using `string.Empty` would case a single allocation instead of multiple, but nowadays the compiler fixes this and it doesn't matter.